src/Entity/Item.php

```diff
@@ -56,6 +66,17 @@ public function __construct(
 	}
 
 	/**
+	 * Returns the id of the entity or null if it does not have one.
+	 *
+	 * @since 5.0
```

Comment:
This is also deceptive, even though in this case the type hint changed. However, since it was made more specific, and since the more specific version would also have worked for the old code, there is no reason for users of this method to note that any change was made here.